packages/sdk-components-react/src/radix-popover.tsx

```diff
@@ -0,0 +1,89 @@
+/* eslint-disable react/display-name */
+// We can't use .displayName until this is merged https://github.com/styleguidist/react-docgen-typescript/pull/449
+
+import * as PopoverPrimitive from "@radix-ui/react-popover";
+import {
+  splitPropsWithWebstudioAttributes,
+  type WebstudioAttributes,
+} from "@webstudio-is/react-sdk";
+
+import {
+  forwardRef,
+  type ElementRef,
+  type ComponentPropsWithoutRef,
+  Children,
+  type ReactNode,
+} from "react";
+
+/**
+ * We don't have support for boolean or undefined nor in UI not at Data variables,
+ * instead of binding on "open" prop we bind variable on a isOpen prop to be able to show Popover in the builder
+ **/
+type BuilderPopoverProps = {
+  isOpen: "initial" | "open" | "closed";
```

Comment:
`is*` convention usually imply boolean value. Maybe openState?

Reply:
Yep will fix in next PR with Dialog everythere